src/postcss-loader.js

```diff
@@ -84,8 +84,11 @@ export default {
             '[name]_[local]' :
             '[name]_[local]__[hash:base64:5]',
           ...options.modules,
-          getJSON(filepath, json) {
+          getJSON(filepath, json, outpath) {
             modulesExported[filepath] = json
+            if (typeof options.modules === 'object' && typeof options.modules.getJSON === 'function') {
+              options.modules.getJSON(filepath, json, outpath)
```

Comment:
`getJSON` may be an `async` function, so you need to `return options.modules.getJSON(...)` here.